Trading/Exchange/coinbase_pro/coinbase_pro_exchange.py

```diff
@@ -58,9 +60,18 @@ async def cancel_order(self, order_id, symbol=None, **kwargs):
     async def get_my_recent_trades(self, symbol=None, since=None, limit=None, **kwargs):
         return self._uniformize_trades(await super().get_my_recent_trades(symbol=symbol,
                                                                           since=since,
-                                                                          limit=limit,
+                                                                          limit=self._fix_limit(limit),
                                                                           **kwargs))
 
+    async def get_open_orders(self, symbol=None, since=None, limit=None, **kwargs) -> list:
+        return await super().get_open_orders(symbol=symbol,
+                                             since=since,
+                                             limit=self._fix_limit(limit),
+                                             params=kwargs)
```

Comment:
There is no param argument in
```python
    async def get_open_orders(self, symbol: str = None, since: int = None,
                              limit: int = None, **kwargs: dict) -> list:
        if self.client.has['fetchOpenOrders']:
            return await self.client.fetch_open_orders(symbol=symbol, since=since, limit=limit, params=kwargs)
        else:
            raise octobot_trading.errors.NotSupported("This exchange doesn't support fetchOpenOrders")
```
it should be `**kwargs`